ajax/ajaxVote.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+include('config/init.php');
+
+
+// if it gets to here, saying an upvote should happen (from the UI standpoint at least)
+//actually wait, p sure it should just like trigger it. like if it gets called on by the UI
+//		js function, then check the db, if can do, do itt
+
+
+
+
+
+
+//if a vote for this PK doesnt exist in DB, create itt
+	//then if request direction up
+		//increase vote in DB
+	//else (aka down)
+		//decrease vote in DB
+
+$exists = doesUserVoteExist($_REQUEST['eventID'], $_REQUEST['userID']);
```

Comment:
You should never trust the user to pass in a userID. With the code as it is now, anyone could forge votes for other users. Instead, require them to be logged in, and get the userID from the session.